sdk/identity/identity/test/internal/node/managedIdentityCredential.spec.ts

```diff
@@ -232,12 +226,12 @@ describe("ManagedIdentityCredential", function() {
       ...mockHttpClient.tokenCredentialOptions
     });
 
-    const promise = credential.getToken("scopes");
+    const clock = sandbox.useFakeTimers();
 
-    imdsMsiRetryConfig.startDelayInMs = 80; // 800ms / 10
+    const promise = credential.getToken("scopes");
 
-    // 800ms -> 1600ms -> 3200ms, results in 6400ms, / 10 = 640ms
-    clock.tick(640);
+    // 800ms -> 1600ms -> 3200ms, results in 6400ms
+    await clock.tickAsync(6400);
```

Comment:
The use of Sinon timers wasn’t properly working here (notice I needed to change the startDelayInMs to 80 milliseconds to make this bearable). The reason was that I was setting `shouldAdvanceTime` to true when defining the clock. I also didn’t know about `tickAsync` until recently.